### PR TITLE
feat(sheet): add documentation

### PR DESCRIPTION
**Pull request by Francois-Esquire**
## Description

This PR adds documentation for the `Sheet` component.

## Detail

The PR adds the `Sheet` component with 5 examples and documentation for usage and API.

## Checklist

- [x] :ok_hand: website updates are Garden Designer approved (add the designer as a reviewer)
- [ ] :black_nib: copy updates are approved (add the content strategist as a reviewer)
- [x] :link: considered opportunities for adding cross-reference URLs (grep for keywords)
- [x] :wheelchair: analyzed via [axe](https://www.deque.com/axe/) and evaluated using VoiceOver
- [x] :memo: tested in Chrome, Firefox, Safari, Edge, and IE11

### CODE REVIEW

File: src/nav/components.yml [hunk 1]
```diff
@@ -48,6 +48,8 @@
           title: Toggle icon button
     - title: Chrome
       id: /components/chrome
+    - title: Sheet
+      id: /components/sheet
```
Comment (Francois-Esquire) [inline src/nav/components.yml:52]: Depending on where the team thinks the Sheet component should be placed in the nav, we can move around accordingly.

File: src/pages/components/sheet.mdx [hunk 2]
```diff
@@ -0,0 +1,169 @@
...
+
+### Sheet.Body
+
+The Sheet Body component is placed in the [`Sheet`](#sheet) component.
+This component requires accessible functionality to make the
+element focusable, if keyboard scrolling is needed.
+
+<Component components={props.data.mdx.components} componentName="sheet.body" />
+
+<PropSheet components={props.data.mdx.components} componentName="sheet.body" />
```
Comment (Francois-Esquire) [inline src/pages/components/sheet.mdx:110]: It seems the any component other than `Sheet` doesn't have a given `Extends HTMLAttributes<HTMLElement>` attributed. Is there something missing on my end on why this is missing?

Comment (Francois-Esquire) [reply src/pages/components/sheet.mdx:110]: Example:
Screen Shot 2022-01-21 at 3 06 20 PM

Comment (jzempel) [reply src/pages/components/sheet.mdx:140]: In its current form, you are responsible for fulfilling the JSDoc API for subcomponents. See https://github.com/zendeskgarden/website/blob/main/src/examples/accordions/components.ts (data) and https://github.com/zendeskgarden/website/blob/main/src/pages/components/accordion.mdx (usage) for details.

With the follow-on to https://github.com/zendeskgarden/react-components/pull/1273, I'm expecting to automate the data gathering for subcomponents, just as it is for top-level element components.

File: src/nav/components.yml (hunk 1 above)
Comment (jzempel) [reply src/nav/components.yml:52]: Let's respect current alphabetical ordering.

File: src/examples/chrome/SheetSize.tsx [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import React, { useState } from 'react';
+import { Sheet } from '@zendeskgarden/react-chrome';
+import { Grid, Row, Col } from '@zendeskgarden/react-grid';
+import { Button } from '@zendeskgarden/react-buttons';
+import { Fieldset, Field, Radio, Label } from '@zendeskgarden/react-forms';
+
+const [col4, col6, px380, px480] = ['4', '6', '380px', '480px'];
```
Comment (jzempel) [inline src/examples/chrome/SheetSize.tsx:14]: It feels like all the following codez would read simpler if the mapping could somehow be expressed up front as a constant. The behavior should be...

Col `size={4}` is Sheet `size="100%"`
Col `size={6}` is Sheet `size="100%"`
Sheet `size="380px"` is Col `size="auto"`
Sheet `size="480px"` is Col `size="auto"`

File: src/examples/chrome/SheetSize.tsx [hunk 4]
```diff
@@ -0,0 +1,83 @@
...
+                </Radio>
+              </Field>
+            ))}
+          </Fieldset>
+        </Col>
+
+        <Col size={size.endsWith('px') ? 9 : Number.parseInt(size, 10)}>
+          <div
+            style={{ width: '100%', height: '500px', display: 'flex', justifyContent: 'flex-end' }}
+          >
```
Comment (jzempel) [inline src/examples/chrome/SheetSize.tsx:45]: Per previous comments, prefer Garden grid layout over extra `div`

File: src/examples/chrome/SheetSize.tsx [hunk 5]
```diff
@@ -0,0 +1,83 @@
...
+  return (
+    <Grid gutters={false} style={{ outline: '1px dotted gray' }}>
+      <Row justifyContent="between">
+        <Col size="auto">
+          <Fieldset style={{ margin: '25px' }}>
+            <Fieldset.Legend>Sheet size</Fieldset.Legend>
+            {[col4, col6, px380, px480].map((sampleSize: string) => (
+              <Field key={`sizes-${sampleSize}`}>
+                <Radio
+                  name="default example"
```
Comment (jzempel) [inline src/examples/chrome/SheetSize.tsx:28]: `name` not needed for a non-submitted layout control

File: src/pages/components/sheet.mdx [hunk 6]
```diff
@@ -0,0 +1,199 @@
...
+
+The Sheet FooterItem component is nested in the [`Sheet.Footer`](#sheetfooter) component.
+
+<Component components={props.data.mdx.components} componentName="sheet.footeritem" />
+
+<PropSheet components={props.data.mdx.components} componentName="sheet.footeritem" />
+
+### Sheet.Close
+
+The Sheet Close button component is placed in the [`Sheet`](#sheet) component.
```
Comment (jzempel) [inline src/pages/components/sheet.mdx:187]: Is it worth identifying how it should be last for consistent tab order?

File: src/pages/components/sheet.mdx [hunk 7]
```diff
@@ -0,0 +1,199 @@
...
+The Sheet Header component is placed in the [`Sheet`](#sheet) component.
+
+<Component components={props.data.mdx.components} componentName="sheet.header" />
+
+<PropSheet components={props.data.mdx.components} componentName="sheet.header" />
+
+### Sheet.Footer
+
+The Sheet Footer component is placed in the [`Sheet`](#sheet) component.
+The component can alternate to compact styling.
```
Comment (jzempel) [inline src/pages/components/sheet.mdx:171]: Please reference other component documentation for use of active voice. Also, something about this "...can alternate" feels a bit off grammatically.

**Review by jzempel — CHANGES_REQUESTED** [review]
Were the examples designed for responsive layout? The Garden website is meant to provide mobile responsive throughout. Please re-check all the examples. I'd recommend using `<Col sm>` for all of the 3-column layouts and then reference code under https://garden.zendesk.com/components/button-group#size for `StyledCol` and `mediaQuery` handling for stacked layouts. Also recommend working through the inline comments first (leaning into Garden layouts), as this should set things up for more consistent grid use.

It also looks like the API doc needs careful content review for passive voice.

**Comment by james-sann** [comment]
> Were the examples designed for responsive layout? The Garden website is meant to provide mobile responsive throughout. Please re-check all the examples. I'd recommend using `<Col sm>` for all of the 3-column layouts and then reference code under https://garden.zendesk.com/components/button-group#size for `StyledCol` and `mediaQuery` handling for stacked layouts. Also recommend working through the inline comments first (leaning into Garden layouts), as this should set things up for more consistent grid use.
> 
> It also looks like the API doc needs careful content review for active vs passive voice.

@jzempel Are you asking the Garden designers to provide responsive design solutions for the component documentation page? If so, where can we find the breakpoints that might help guide @Francois-Esquire? @jon-kerwin were you aware of these? Happy to help. Just need the process shown to me!

**Comment by jzempel** [comment]
> Are you asking the Garden designers to provide responsive design solutions for the component documentation page?

No. In the past, these have been worked out on-the-fly during implementation. However, if we feel that a process adaptation is necessary, breakpoint values can be found here: https://garden.zendesk.com/components/theme-object

File: src/pages/components/sheet.mdx [hunk 8]
```diff
@@ -0,0 +1,199 @@
...
+import SheetPlacementCode from '!!raw-loader!../../examples/chrome/SheetPlacement.tsx';
+
+<CodeExample code={SheetPlacementCode}>
+  <SheetPlacement />
+</CodeExample>
+
+### Size
+
+Sheets default to 380px in width. You can customise the width based on your intended
+experience to any size using pixels or our grid system.
```
Comment (diana-oum) [inline src/pages/components/sheet.mdx:85]: Sorry if this was discussed as I've been out of the loop but looking at the Figma file, "our grid system" is linked to https://garden.zendesk.com/components/grid

File: src/examples/chrome/SheetDefault.tsx [hunk 9]
```diff
@@ -0,0 +1,80 @@
...
+            >
+              <Label>Show Sheet</Label>
+            </Toggle>
+          </Field>
+        </Col>
+
+        <Col size={8}>
+          <div style={{ width: '100%', height: '500px', display: 'flex' }}>
+            {/* the div below represents main content sharing space with the Sheet in a container */}
+            <div style={{ flexGrow: 1 }} />
```
[the remark this answers is not in the document]
Comment (Francois-Esquire) [reply src/examples/chrome/SheetDefault.tsx:37]: Sounds good, will remove the divs from the examples.

File: src/examples/chrome/SheetFooters.tsx [hunk 10]
```diff
@@ -0,0 +1,66 @@
...
+      <Col size={4}>
+        <Sheet isOpen size="100%" style={surroundingBorderStyle}>
+          <Sheet.Close />
+        </Sheet>
+      </Col>
+
+      <Col size={4}>
+        <Sheet isOpen size="100%" style={surroundingBorderStyle}>
+          {/* div element below emulates Sheet.Body in its absence */}
+          <div style={{ flexGrow: 1 }} />
```
[the remark this answers is not in the document]
Comment (Francois-Esquire) [reply src/examples/chrome/SheetFooters.tsx:32]: Good call, will update this.

File: src/examples/chrome/SheetFooters.tsx [hunk 11]
```diff
@@ -0,0 +1,66 @@
...
+ */
+
+import React from 'react';
+import { Sheet } from '@zendeskgarden/react-chrome';
+import { Grid, Row, Col } from '@zendeskgarden/react-grid';
+import { Anchor, Button } from '@zendeskgarden/react-buttons';
+
+const surroundingBorder = '1px dotted gray';
+const surroundingBorderStyle = {
+  borderRight: surroundingBorder,
```
[the remark this answers is not in the document]
Comment (Francois-Esquire) [reply src/examples/chrome/SheetFooters.tsx:15]: Yes good point, missed this originally, will update.

[the remark this answers is not in the document]
Comment (mtomcal) [reply src/examples/chrome/SheetFooters.tsx:15]: Ill echo @jzempel on his other comment https://github.com/zendeskgarden/website/pull/337/files#r790861329 about replacing this object with a reusable styled component.

File: src/examples/chrome/SheetFooters.tsx [hunk 12]
```diff
@@ -0,0 +1,66 @@
...
+      </Col>
+
+      <Col size={4}>
+        <Sheet isOpen size="100%" style={surroundingBorderStyle}>
+          {/* div element below emulates Sheet.Body in its absence */}
+          <div style={{ flexGrow: 1 }} />
+
+          <Sheet.Footer>
+            {/* margin-left was set to 0 due to column width wrapping the two buttons */}
+            <Sheet.FooterItem style={{ marginLeft: '0' }}>
```
[the remark this answers is not in the document]
Comment (Francois-Esquire) [reply src/examples/chrome/SheetFooters.tsx:51]: Agreed, this was something that came to mind while working on the documentation.

File: src/examples/chrome/SheetFooters.tsx [hunk 13]
```diff
@@ -0,0 +1,66 @@
...
+      </Col>
+
+      <Col size={4}>
+        <Sheet isOpen size="100%" style={surroundingBorderStyle}>
+          {/* div element below emulates Sheet.Body in its absence */}
+          <div style={{ flexGrow: 1 }} />
+
+          <Sheet.Footer isCompact>
+            <Sheet.FooterItem>
+              <Anchor>Anchor</Anchor>
```
[the remark this answers is not in the document]
Comment (Francois-Esquire) [reply src/examples/chrome/SheetFooters.tsx:36]: Agreed, it is off center, will update.

File: src/pages/components/sheet.mdx (hunk 8 above)
Comment (Francois-Esquire) [reply src/pages/components/sheet.mdx:85]: Good call, I'll update and link to Grid.

File: src/pages/components/sheet.mdx (hunk 6 above)
Comment (Francois-Esquire) [reply src/pages/components/sheet.mdx:187]: Good call on this.

File: src/pages/components/sheet.mdx (hunk 7 above)
Comment (Francois-Esquire) [reply src/pages/components/sheet.mdx]: Will update this and some of the text.

**Comment by Francois-Esquire** [comment]
> > Were the examples designed for responsive layout? The Garden website is meant to provide mobile responsive throughout. Please re-check all the examples. I'd recommend using `<Col sm>` for all of the 3-column layouts and then reference code under https://garden.zendesk.com/components/button-group#size for `StyledCol` and `mediaQuery` handling for stacked layouts. Also recommend working through the inline comments first (leaning into Garden layouts), as this should set things up for more consistent grid use.
> > It also looks like the API doc needs careful content review for active vs passive voice.
> 
> @jzempel Are you asking the Garden designers to provide responsive design solutions for the component documentation page? If so, where can we find the breakpoints that might help guide @Francois-Esquire? @jon-kerwin were you aware of these? Happy to help. Just need the process shown to me!

Thanks @james-sann & @jzempel  for your help with thinking through the responsive design! I've updated the Sheet page with what we discussed offline and through the review. Please do let me know if I missed anything.

File: src/pages/components/sheet.mdx [hunk 14]
```diff
@@ -0,0 +1,201 @@
...
+</Sheet>
+```
+
+### Sheet
+
+<Component components={props.data.mdx.components} componentName="sheet" />
+
+The `Sheet` component provides state and accessibility attributes to its children.
+
+<PropSheet components={props.data.mdx.components} componentName="sheet" />
```
Comment (jzempel) [inline src/pages/components/sheet.mdx:131]: It looks like there are some stray asterisks following prop descriptions that need to be cleaned-up as a TODO on the `react-components` codebase.

**Review by jzempel — CHANGES_REQUESTED** [review]

File: src/nav/components.yml (hunk 1 above)
Comment (jzempel) [reply src/nav/components.yml]: btw, we can see here that "Color picker" and "Color swatch" are out of order – might be nice to sweep those up, too.

**Review by james-sann — CHANGES_REQUESTED** [review]
@Francois-Esquire Great work! Thanks for updating the responsive version.

Within the Placement section, we discussed making the container 90% on mobile viewports so there was a clear delineation of placing at the start or not. I couldn't see this reflected in the build. Just wondering if this was consciously addressed or not 🙌

File: src/examples/chrome/SheetFooters.tsx [hunk 15]
```diff
@@ -0,0 +1,66 @@
...
+const surroundingBorder = '1px dotted gray';
+const surroundingBorderStyle = {
+  borderRight: surroundingBorder,
+  borderTop: surroundingBorder,
+  borderBottom: surroundingBorder
+};
+
+const Example = () => (
+  <Grid gutters="sm">
+    <Row style={{ height: '525px' }}>
```
[the remark this answers is not in the document]
Comment (jzempel) [reply src/examples/chrome/SheetFooters.tsx]: @mtomcal we can't do shared components for examples, as they must successfully open/execute in the fixed Codesandbox environment.

File: src/pages/components/sheet.mdx [hunk 16]
```diff
@@ -0,0 +1,201 @@
...
+
+### Sheet.Body
```
[the remark this answers is not in the document]
Comment (Francois-Esquire) [reply src/pages/components/sheet.mdx:133]: Good catch, will update this.

File: src/pages/components/sheet.mdx [hunk 17]
```diff
@@ -0,0 +1,201 @@
...
+
+<Component components={props.data.mdx.subcomponents} componentName="sheet.body" />
+
+Nest `Sheet.Body` in the [`Sheet`](#sheet) component.
+`Sheet.Body` requires additional accessible functionality to make the
+element focusable for keyboard scrolling.
```
[the remark this answers is not in the document]
Comment (mtomcal) [reply src/pages/components/sheet.mdx:139]: Should this ☝️ be recorded somewhere about documentation guidelines? TIL

File: src/examples/chrome/SheetDefault.tsx [hunk 18]
```diff
@@ -0,0 +1,98 @@
...
+      box-sizing: content-box;
+      width: 100%;
+    }
+
+    /* stylelint-disable */
+    & > aside > div {
+      min-width: 100%;
+    }
+    /* stylelint-enable */
+  }
```
[the remark this answers is not in the document]
Comment (Francois-Esquire) [reply src/examples/chrome/SheetDefault.tsx:35]: I don't think `box-sizing` is needed here, I will remove it.

File: src/pages/components/sheet.mdx (hunk 14 above)
Comment (Francois-Esquire) [reply src/pages/components/sheet.mdx:131]: Raised [PR #1292](https://github.com/zendeskgarden/react-components/pull/1292) to take care of this.

**Comment by Francois-Esquire** [comment]
> @Francois-Esquire Great work! Thanks for updating the responsive version.
> 
> Within the Placement section, we discussed making the container 90% on mobile viewports so there was a clear delineation of placing at the start or not. I couldn't see this reflected in the build. Just wondering if this was consciously addressed or not 🙌

@james-sann I missed this frankly speaking, more than happy to update! Before I do, what size do we want to set the sheet as the baseline? Should we follow the designs to the T here (ranged from 496px and up) or otherwise?

> Once the factor is bigger than 10, it feels like there are diminishing returns for leveraging `space.base`. For example, if the base were changed to 5, you probably don't want the height to change from 480 to 600px. Same for three – 480 down to 360px.

@jzempel agree, happy to update with a bit more direction here. This is probably related to the above, we could use a set pixel size here for all the examples (not just the headers/footers examples) since I noticed on IE11, the height was completely flattened here:

Screen Shot 2022-02-07 at 5 28 19 PM

File: src/pages/components/sheet.mdx (hunk 17 above)
[the remark this answers is not in the document]
Comment (Francois-Esquire) [reply src/pages/components/sheet.mdx]: @jzempel I removed the backticks. I'll reach out to @smritimadan here for more guidance.

@mtomcal you raise a good point here, worth recording down.

File: src/examples/chrome/SheetPlacement.tsx [hunk 19]
```diff
@@ -0,0 +1,91 @@
...
+            </Toggle>
+          </StyledField>
+        </Col>
+      </Row>
+
+      <StyledRow justifyContent={placement}>
+        <StyledSheet isOpen placement={placement}>
+          <Sheet.Header>
+            <Sheet.Title>Placement</Sheet.Title>
+            <Sheet.Description>From side to side</Sheet.Description>
```
Comment (Francois-Esquire) [inline src/examples/chrome/SheetPlacement.tsx:60]: Needed to deviate from the original content here due to axe testing and the need for unique `aria-labelledby` & `aria-describedby` values. Will need a content check here, and happy to update to what we want.

File: src/examples/chrome/SheetSize.tsx [hunk 20]
```diff
@@ -0,0 +1,97 @@
...
+          </StyledFieldset>
+        </Col>
+      </Row>
+
+      <StyledRow justifyContent="end">
+        <Col size={size.cols}>
+          <Sheet isOpen isAnimated={false} size={size.sheet}>
+            <Sheet.Header>
+              <Sheet.Title>Size</Sheet.Title>
+              <Sheet.Description>Sizing the Sheet</Sheet.Description>
```
Comment (Francois-Esquire) [inline src/examples/chrome/SheetSize.tsx:65]: The same update here (as above comment) due to axe raising this as an issue.

File: src/examples/chrome/SheetFooters.tsx [hunk 21]
```diff
@@ -0,0 +1,79 @@
...
+import { Sheet } from '@zendeskgarden/react-chrome';
+import { Row, Col } from '@zendeskgarden/react-grid';
+import { Anchor, Button } from '@zendeskgarden/react-buttons';
+import { getColor, mediaQuery } from '@zendeskgarden/react-theming';
+
+const StyledCol = styled(Col)`
+  height: ${props => props.theme.space.base * 120}px;
+
+  ${props => mediaQuery('down', 'sm', props.theme)} {
+    height: ${props => props.theme.space.base * 80}px;
```
[the remark this answers is not in the document]
Comment (jzempel) [reply src/examples/chrome/SheetFooters.tsx:19]: My point here was that we should just use:

```
height: 480px;

${props => mediaQuery('down', 'sm', props.theme)} {
  height: 320px;
```

File: src/examples/chrome/SheetFooters.tsx [hunk 22]
```diff
@@ -0,0 +1,97 @@
...
+`;
+
+const Example = () => (
+  <Row>
+    <StyledCol md>
+      <StyledSheet
+        isOpen
+        size="100%"
+        aria-describedby=""
+        aria-labelledby=""
```
Comment (jzempel) [inline src/examples/chrome/SheetFooters.tsx:47]: Per conversation, please remove all empty `aria-describedby/labelledby`. They don't contribute to a clear understanding of expected usage and are indicators that the component itself should be improved to add valid attributes only when `Title` and/or `Description` exist.

File: src/examples/chrome/SheetPlacement.tsx (hunk 19 above)
Comment (jzempel) [reply src/examples/chrome/SheetPlacement.tsx:60]: To be fair, the axe warnings here are a trailing concern. The leading concern is aria attribute management on the component itself. I would advocate for allowing the warnings to show here (and honoring the content design), as they are representative of an underlying component issue.

File: src/examples/chrome/SheetSize.tsx (hunk 20 above)
Comment (jzempel) [reply src/examples/chrome/SheetSize.tsx:65]: ditto comment reply.

**Review by jzempel — CHANGES_REQUESTED** [review]

**Review by james-sann — CHANGES_REQUESTED** [review]
@Francois-Esquire Just checked the RTL versions of the **Headers** and **Footers** examples, and they look a tad broken (Solid grey border-left is missing?)

Within the **Footers** section on RTL, the footer buttons are also stacking. They don't appear stacked on the LTR option.

image

File: src/examples/chrome/SheetFooters.tsx (hunk 22 above)
Comment (Francois-Esquire) [reply src/examples/chrome/SheetFooters.tsx]: This has been updated!

**Comment by Francois-Esquire** [comment]
> @Francois-Esquire Just checked the RTL versions of the **Headers** and **Footers** examples, and they look a tad broken (Solid grey border-left is missing?)
> 
> Within the **Footers** section on RTL, the footer buttons are also stacking. They don't appear stacked on the LTR option.
> 
> image

Thanks for pointing this out @james-sann ! I made some updates that addresses these issues. Let me know if anything else arises.

**Review by jzempel — APPROVED** [review]

File: src/pages/components/sheet.mdx [hunk 23]
```diff
@@ -0,0 +1,200 @@
...
+Nest it within a [Header](#sheetheader) component.
+
+<PropSheet components={props.data.mdx.subcomponents} componentName="sheet.description" />
+
+#### Sheet.Header
+
+<Component components={props.data.mdx.subcomponents} componentName="sheet.header" />
+
+Nest a Header within the [Sheet](#sheet) component. If neither [Title][#sheettitle] nor
+[Description][#sheetdescription] are used, the Header should be omitted.
```
Comment (mtomcal) [inline src/pages/components/sheet.mdx:166]:
```suggestion
Nest a Header within the [Sheet](#sheet) component. If neither [Title](#sheettitle) nor
[Description](#sheetdescription) are used, the Header should be omitted.
```
Found this:
Screen Shot 2022-02-14 at 3 43 41 PM

**Review by mtomcal — CHANGES_REQUESTED** [review]
Quick fix before approval

**Review by james-sann — APPROVED** [review]
Looks great Mike! Approved!

**Review by hzhu — APPROVED** [review]

File: src/pages/components/sheet.mdx (hunk 23 above)
Comment (Francois-Esquire) [reply src/pages/components/sheet.mdx]: Ahh good find! Thank you.

**Review by exelarios — APPROVED** [review]

**Review by mtomcal — APPROVED** [review]

File: src/pages/components/sheet.mdx [hunk 24]
```diff
@@ -0,0 +1,200 @@
...
+import SheetPlacement from '../../examples/chrome/SheetPlacement.tsx';
+import SheetPlacementCode from '!!raw-loader!../../examples/chrome/SheetPlacement.tsx';
+
+<CodeExample code={SheetPlacementCode}>
+  <SheetPlacement />
+</CodeExample>
+
+### Size
+
+Sheets default to 380px in width. You can customise the width based on your intended
```
Comment (diana-oum) [inline src/pages/components/sheet.mdx:85]: @Francois-Esquire Just caught this now. "customise" --> "customize" 
We use American English spelling. It's mentioned in my [How to write for Garden guidelines](https://zendesk.atlassian.net/wiki/spaces/GARDEN/pages/5372510361/How+to+write+for+Garden) which got published after we designed this documentation but going forward I'll be really pushing these guidelines for both designers and engineers so we can all catch them earlier in the process. cc: @james-sann @jon-kerwin 

I was able to catch most of these things when in it was in the documentation phase but sometimes they still slip through...

**Review by diana-oum — APPROVED** [review]
"customise" needs to be "customize" but other than that, approved!

File: src/pages/components/sheet.mdx (hunk 24 above)
Comment (Francois-Esquire) [reply src/pages/components/sheet.mdx]: Ahh I missed that as well! Good catch Diana, thank you!

**Pull request merged**